app/models/vm_or_template.rb

```diff
@@ -130,7 +130,6 @@ class VmOrTemplate < ApplicationRecord
   virtual_column :os_image_name,                        :type => :string,     :uses => [:operating_system, :hardware]
   virtual_column :platform,                             :type => :string,     :uses => [:operating_system, :hardware]
   virtual_column :vendor_display,                       :type => :string
-  virtual_column :v_host_vmm_product,                   :type => :string,     :uses => :host
```

Comment:
column is staying, and is added below.

I'll add a test to ensure this column is still available.